### PR TITLE
RELATED: RAIL-3910 Adding info about plugin validation

### PR DESCRIPTION
**Pull request by evgeniabogranskaya**
…rsion of the Dashboard component

### CODE REVIEW

File: docs/18_dashboard_plugins.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ copyright: (C) 2007-2021 GoodData Corporation
 id: dashboard_plugins
 ---
 
-Dashboard plugins allow developers to create and integrate custom code into the KPI Dashboards component. With the plugins, you
+Dashboard plugins allow developers to create and integrate custom code into the Dashboards component. With the plugins, you
```
Comment (no23reason) [inline docs/18_dashboard_plugins.md:8]: We call it Dashboard component (singular).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/952/ : SUCCESS in 1m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/632/ : SUCCESS in 7s

File: docs/18_dashboard_plugins.md (hunk 1 above)
Comment (evgeniabogranskaya) [reply docs/18_dashboard_plugins.md]: Fixed!

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR484

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/954/ : SUCCESS in 1m 28s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/189/ : SUCCESS in 43s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/84/ : SUCCESS in 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/279/ : SUCCESS in 20s

**Pull request merged**